pkg/polaris/ctl_polaris.go

```diff
@@ -120,45 +120,64 @@ func (ctl *CRSpecBuilderFromCobraFlags) SetPredefinedCRSpec(specType string) err
+	cmd.Flags().StringVar(&ctl.GCPServiceAccount, "gcp-service-account-path", ctl.GCPServiceAccount, "Absolute path to the Google Cloud service account given to pull images")
+	cmd.Flags().StringVarP(&ctl.polarisLicensePath, "polaris-license-path", "", ctl.polarisLicensePath, "Absolute path to the given Polaris license")
+	cmd.Flags().StringVarP(&ctl.coverityLicensePath, "coverity-license-path", "", ctl.coverityLicensePath, "Absolute path to the given Coverity license")
 
+	// smtp related flags
 	cmd.Flags().StringVar(&ctl.SMTPHost, "smtp-host", ctl.SMTPHost, "SMTP host")
 	cmd.Flags().IntVar(&ctl.SMTPPort, "smtp-port", ctl.SMTPPort, "SMTP port")
 	cmd.Flags().StringVar(&ctl.SMTPUsername, "smtp-username", ctl.SMTPUsername, "SMTP username")
 	cmd.Flags().StringVar(&ctl.SMTPPassword, "smtp-password", ctl.SMTPPassword, "SMTP password")
 	cmd.Flags().StringVar(&ctl.SMTPSenderEmail, "smtp-sender-email", ctl.SMTPSenderEmail, "SMTP sender email")
 	cmd.Flags().StringVar(&ctl.SMTPTlsMode, "smtp-tls-mode", string(GetPolarisDefault().PolarisDBSpec.SMTPDetails.TLSMode), "SMTP TLS mode [disable|try-starttls|require-starttls|require-tls]")
 	cmd.Flags().StringVar(&ctl.SMTPTlsTrustedHosts, "smtp-trusted-hosts", ctl.SMTPTlsTrustedHosts, "Whitespace separated list of trusted hosts")
-	cmd.Flags().BoolVar(&ctl.SMTPTlsIgnoreInvalidCert, "smtp-tls-insecure", false, "Accept invalid certificates")
+	cmd.Flags().BoolVar(&ctl.SMTPTlsIgnoreInvalidCert, "enable-smtp-tls-insecure", false, "Enable accepting invalid certificates")
 
-	cmd.Flags().StringVarP(&ctl.organizationProvisionOrganizationDescription, "organization-description", "", ctl.organizationProvisionOrganizationDescription, "Organization description")
+	// domain-name specific flags
+	cmd.Flags().StringVar(&ctl.EnvironmentDNS, "domain-name", ctl.EnvironmentDNS, "Fully qualified domain name (FQDN)")
+
+	// postgres specific flags
+	// these flags are specific for an external managed postgres
+	cmd.Flags().StringVar(&ctl.PostgresHost, "postgres-host", ctl.PostgresHost, "Postgres host")
+	cmd.Flags().IntVar(&ctl.PostgresPort, "postgres-port", ctl.PostgresPort, "Postgres port")
+	cmd.Flags().StringVar(&ctl.PostgresSSLMode, "postgres-ssl-mode", string(GetPolarisDefault().PolarisDBSpec.PostgresDetails.SSLMode), "Postgres ssl mode [disable|require]")
+	cmd.Flags().StringVar(&ctl.PostgresUsername, "postgres-username", ctl.PostgresUsername, "Postgres username")
+	// if using in-cluster containerized Postgres, then currently we require "enable-postgres-container", "postgres-password" and optionally "postgres-size"
+	// [TODO: make the above point clear to customers]
+	cmd.Flags().StringVar(&ctl.PostgresPassword, "postgres-password", ctl.PostgresPassword, "Postgres password")
+	cmd.Flags().BoolVar(&ctl.PostgresInternal, "enable-postgres-container", GetPolarisDefault().PolarisDBSpec.PostgresDetails.IsInternal, "Enable in-cluster containerized postgres backed by a persistent volume (Not recommended for production usage)")
```

Comment:
I'm thinking about the description. This sounds like the flag gives the user the ability to put postgres into their cluster. As opposed to the Polaris instance will be deployed with it's own internal postgres. Maybe change the description to "If true, Polaris uses its own internal postgres instance."

Reply:
I don't think "If true, Polaris uses its own internal postgres instance." clarifies it any-more either :-) Frankly, I don't like the term internal because it's ambiguous, "internal" to what? The usage of the word "in-cluster" is to indicate that postgres will run in their k8s cluster alongside other polaris pods.  How about, something like "If true, Polaris will use an in-cluster postgres backed by a persistent volume"

Reply:
done :)